uber/utils.py

```diff
@@ -1044,41 +1048,32 @@ def groups(self):
     def stripe_transaction(self):
         return self._stripe_transaction
 
-    def charge_cc(self, session, token):
+    def create_stripe_intent(self, session):
+        log.debug('PAYMENT: !!! attempting to charge {} cents for {}', self.amount, self.description)
         try:
-            log.debug('PAYMENT: !!! attempting to charge stripeToken {} {} cents for {}',
-                      token, self.amount, self.description)
-
-            self.response = stripe.Charge.create(
-                card=token,
+            stripe_intent = stripe.PaymentIntent.create(
+                payment_method_types=['card'],
+                amount=self.amount,
                 currency='usd',
-                amount=int(self.amount),
                 description=self.description,
-                receipt_email=self.receipt_email
```

Comment:
Hm, we should make sure Stripe still sends out receipt emails for this workflow.